src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -212,6 +212,14 @@ class CommandRequestImpl : public CommandImpl,
   bool HasDisallowedParams() const;
 
   /**
+   * @brief Checks if file name contains invalid symbols e.g. '/'
```

Comment:
@AByzhynar according to [Wikipedia](https://en.wikipedia.org/wiki/Filename) the only prohibited character in file names for Unix-like systems is '/' so this function seems correct